### DISCUSSION TITLE
Povratna informacija na prijavu projekta

### DISCUSSION THREAD

**Discussion by MarkoMijac** [foivz/pi23-igadzic-rhabdija-kipsa-aslavic#1] [General]
Poštovani kolege,

Oznaka vašeg tima je: PI2313
Mentor: Marko Mijač

molim da do kraja vikenda prijavite vašu temu, i to tako da svaki član tima ima barem 3 funkcionalnosti. Također, prijavu napravite u master grani.

Lijep pozdrav!

**Comment by MarkoMijac** [comment]
Poštovani kolege,

tema vašeg projekta je u redu. Povratne informacije na pojedinačne funkcionalnosti su sljedeće:
F01 - U okviru ove funkcionalnosti između ostalog korisnicima (kupcima) generirajte člansku karticu sa QR kodom koji ih identificira.
F02 - Neka se zaposlenici logiraju preko korisničkog imena i lozinke, a kupci jednostavnim skeniranjem QR koda.
F03 - Početna stranica nije funkcionalnost.
F04 - Ovo je zapravo prikaz kataloga proizvoda, koji između ostalog ima i prikaz stanja zaliha.
F05 - Konobar neka ima mogućnost pregleda narudžbi po stolovima, te izradu računa na temelju odabrane narudžbe.
F06 i F07 su jedna funkcionalnost
F08 - Ovdje napravite funkcionalnost naručivanja robe, koja uključuje izradu i ispis narudžbenice.
F09 - ovo je već uključeno u prethodnoj funkcionalnosti
F10 - ovo je doslovno jedan atribut u tablici proizvoda, to nije zasebna funkcionalnost.
F11 - ova funkcionalnost nema smisla za kafić.
F12 - ok

Uklonite funkcionalnosti za koje sam naveo da nemaju smisla, te nadopunite sa novim smislenim funkcionalnostima. Svaki član tima bi trebao imati barem 3 funkcionalnosti. Evo nekoliko ideja: (1) izrada narudžbe od strane kupca na stolu za kojim sjedi, (2) definiranje i rezervacija stolova, (3) provedba inventure pića...

Lijep pozdrav!

**Comment by geki7** [comment]
Poštovani profesore, 

Napravili smo izmjene funkcionalnosti koje ste predložili i dodali još neke promjene za koje mislimo da su dobre. Možemo li dobiti Vašu provratnu informaciju o našim izmjenama. 

Lijep pozdrav.

**Reply by MarkoMijac** [reply]
Poštovani kolege,

* Generiranje QR koda treba biti dio funkcionalnosti F01, a ne zasebna funkcionalnost. Dakle, za F03 smislite nešto drugo.

Lijep pozdrav!
